### PR TITLE
Add "Connecting your kit" docs

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Attempting to keep history from https://github.com/sourcebots/sbot/tree/90f6bc7780794932e4e2787b9d160ae1abe1329f and https://docs.sourcebots.co.uk/en/latest/kit/index.html

Copies the "Connecting your kit" docs, with amendments.

I also added a dedicated page for CamCons. Whilst it's a very tiny page with little on it, it's useful to reference to for us and competitors as they're unlikely to have heard of them.

### CODE REVIEW

File: docs/tutorials/connecting-your-kit.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+11. At this point, check that everything is connected up correctly (it
+    may be helpful to ask a facilitator to check that all cables
+    are connected properly).
+12. Connect the Power Board to one of the blue LiPo batteries by
+    plugging the yellow connector on the cable connected to the Power
+    Board into its counterpart on the battery.
+13. If there is not one plugged in already, a loop of wire should be
+    connected to the socket beneath the On|Off switch. Check that the
+    Power Board works by pressing the On|Off switch and checking that
+    the bright LED on the Raspberry Pi comes on green.
```
Comment (WillB97) [inline docs/tutorials/connecting-your-kit.md:118]: This should be a picture.

Comment (sedders123) [reply docs/tutorials/connecting-your-kit.md:118]: https://github.com/srobo/docs/blob/main/images/content/kit/kit_assembly.svg

File: docs/assets/img/assembly/kit-assembly.svg
Comment (RealOrangeOne) [inline docs/assets/img/assembly/kit-assembly.svg:1]: Can someone please check that the wire between the power board and pi are the correct way around.

Comment (RealOrangeOne) [reply docs/assets/img/assembly/kit-assembly.svg:1]: We're going to do this when we're on-site.

**Review by sedders123 — APPROVED** [review]

**Pull request merged**